src/main/java/org/openstreetmap/atlas/geography/atlas/Atlas.java

```diff
@@ -185,6 +185,19 @@
      */
     Iterable<AtlasEntity> entities();
 
+    /**
+     * Return all the {@link AtlasEntity}s of a specific type
+     *
+     * @param type
+     *            The type to restrain to
+     * @param memberClass
+     *            The class of the member
+     * @param <M>
+     *            The AtlasEntity type
+     * @return All the {@link AtlasEntity}s of a specific type
+     */
+    <M extends AtlasEntity> Iterable<M> entities(ItemType type, Class<M> memberClass);
```

Comment:
Nit: One quick test in `BareAtlasTest` for this?

Reply:
Added!